vmngclient/model/configuration/feature_profile/sdwan/transport/cellular_controller.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+from typing import Union
+
+from pydantic import Field
+
+from vmngclient.api.configuration_groups.parcel import Default, Global, Parcel, Variable
+
+
+class ConfigTypeValue(Enum):
+    NON_E_SIM = "non-eSim"
+
+
+class ControllerConfig(Parcel):
+    id: Union[Variable, Global[str]] = Field(min_length=1, max_length=5, description="Cellular ID")
+    slot: Union[Variable, Global[int], Default[int], None] = Field(
+        default=None, description="Set primary SIM slot", ge=0, le=1
+    )
+    maxRetry: Union[Variable, Global[int], Default[None], None] = Field(
```

Comment:
use snake case field name

Reply:
Renamed to snake_case. Add `validation_alias` and `serialization_alias`.